### ISSUE TITLE
Determine how to handle text spans, divisions, fonts, styles

### ISSUE THREAD

**Issue by joeberkovitz** [w3c/mnx#11]
MusicXML provides for multiple elements with different styling to be combined into a single direction with its own text flow, through successive `<words>` and/or `<direction-type>` elements.
 There are ways of arranging for both `<div>`-like and `<span>`-like behavior (to put this in HTML terms) although the MusicXML schema isn't crystal clear on when elements get strung together horizontally vs. vertically.

Since CWMNX is more explicitly aligned with web standards, CWMNX should perhaps attempt a way of organizing and styling spans and blocks that is more in line with the way that HTML handles these concepts.

**Comment by siennamw** [comment]
Yes, let's look to HTML and CSS for solutions.  No need to reinvent the wheel.

Contributors unfamiliar with HTML and CSS might be interested in [this HTML tutorial](http://htmldog.com/guides/html/) and [this CSS tutorial](http://htmldog.com/guides/css/).

**Comment by dhoernel** [comment]
The current MNX draft specification is built on the achievements that have been reached with MusicXML. But it also introduces some new design principles which are very well summarized here:
https://w3c.github.io/mnx/specification/#design-notes

One is about "Separation of semantic concerns from presentation/interpretation concerns". I think it is important to follow it in order to get the robustness we are trying to achieve.

CSS has the advantage to be a well-established standard. It could be used to create a uniform look, e.g. for coloring all notes in a specified voice, but style could also be attributed to single elements, e.g. for horizontally shifting a note with default-x.

One aspect: At present when importing a MusicXML score which has not been created by your own program, you will probably read in most semantic elements and attributes which are essential to correctly display the score. But you will also ignore or partially ignore some presentation attributes like default-x or interpretation attributes because your layout engine differs from the layout engine of the program that wrote the MusicXML, or because you are interested in displaying the score, but not in playing it (or vice versa). This means that you have to know exactly what attributes are semantic or presentation/interpretation attributes. What I would find appealing is that I do not have to know that when importing a score, but I can simple consider/ignore the "levels" (semantics/presentation/interpretation) which I am interested in or not. For example, when I do not want to colorize notes, I simply ignore the corresponding styling directive, otherwise I am looking for the elements addressed by the styling and apply it to them.
